Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -92,27 +106,30 @@ public class Klaviyo: NSObject  {
      
      - Parameter apiKey: string representation of the Klaviyo API Key
      */
+    @objc
```

Comment:
Is it sufficient to just mark the class as open to Objective-C instead of marking each method?

Reply:
I'm curious about this too, I would assume the decorator on the class would scope all contained methods too

Reply:
You need it on both I believe. https://developer.apple.com/forums/thread/129282